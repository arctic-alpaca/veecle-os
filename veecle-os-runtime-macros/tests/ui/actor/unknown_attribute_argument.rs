#[veecle_os_runtime_macros::actor(foo = "bar")]
async fn macro_test_actor() -> std::convert::Infallible {
    unreachable!("We only care about the code compiling.");
}

fn main() {}
