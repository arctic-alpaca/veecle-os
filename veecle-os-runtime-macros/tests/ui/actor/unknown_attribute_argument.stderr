error: unknown attribute argument
 --> tests/ui/actor/unknown_attribute_argument.rs:1:35
  |
1 | #[veecle_os_runtime_macros::actor(foo = "bar")]
  |                                   ^^^^^^^^^^^
